index.js

```diff
@@ -338,6 +338,11 @@
   //  Options :: Type
   var Options = $.RecordType({checkTypes: $.Boolean, env: $.Array($.Any)});
 
+  //  toArray :: Foldable f => f a -> Array a
```

Comment:
Please move this even further up the file. Just before `toObject` would be ideal.

Reply:
On my way!